shims/npm

```diff
@@ -3,21 +3,18 @@
 set -eu
 set -o pipefail
 
-# This script wraps npm so to run `asdf reshim` after global installs and uninstalls
+# This script wraps npm so to run `rtx reshim` after global installs and uninstalls
 # Any other cases are passed-through to npm
 
 this_dir=$(dirname "${BASH_SOURCE[0]}")
 plugin_name=$(basename "$(dirname "$this_dir")")
 
 this_dir=$(cd "$this_dir" && pwd -P) # Normalizes the directory
 
-asdf_data_dir=$(cd "${ASDF_DATA_DIR:-$HOME/.asdf}" && pwd -P)
-asdf_shims_dir="$asdf_data_dir/shims"
-
-plugin_dir="$asdf_data_dir/plugins/$plugin_name"
+plugin_dir="${RTX_DATA_DIR}/plugins/$plugin_name"
```

Comment:
Hi, I'm getting the following error since this change:

```
# npm
/home/patrick/.local/share/rtx/plugins/nodejs/shims/npm: line 14: RTX_DATA_DIR: unbound variable
```

Reply:
should be fixed now

Reply:
Can confirm that, thanks!